lib/features/manage_account/data/network/manage_account_api.dart

```diff
@@ -142,4 +147,37 @@ class ManageAccountAPI {
 
     return result?.list.first.rules ?? <TMailRule>[];
   }
+
+  Future<List<TMailRule>> updateListTMailRule(
```

Comment:
we can use the same API for edit, create, delete. But in `datasource` and `repository`, please define each operator separately